oe_corporate_blocks.post_update.php

```diff
@@ -12,6 +12,40 @@
  */
 function oe_corporate_blocks_post_update_20001() {
   // Delete the config.
-  \Drupal::configFactory()->getEditable('oe_corporate_blocks.data.site_switcher')
+  \Drupal::configFactory()
+    ->getEditable('oe_corporate_blocks.data.site_switcher')
     ->delete();
 }
+
+/**
+ * Update corporate footer.
+ */
+function oe_corporate_blocks_post_update_20002(&$sandbox) {
+  $config = \Drupal::configFactory()->getEditable('oe_corporate_blocks.data.footer');
+  $config->set('about_ec_links', [
+    [
+      'href' => 'https://ec.europa.eu/info/index_en',
+      'label' => 'European Commission website',
+    ],
+  ]);
+  $config->save();
+  // We have to update configuration also for translated configs manually.
```

Comment:
Isn't this gonna be imported when running the configuration import Drush command that we run after every deployment?

Reply:
Described